hedera-node/configuration/previewnet/bootstrap.properties

```diff
@@ -3,7 +3,4 @@ contracts.chainId=297
 bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
-autoCreation.enabled=true
-lazyCreation.enabled=true
-cryptoCreateWithAliasAndEvmAddress.enabled=true
\ No newline at end of file
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```

Comment:
are we still interested in turning on all sidecars everywhere? (e.g., as seen in #4708) - in that case this line shouldn't be here, it should be in `hedera-node/hedera-mono-services/src/main/resources/bootstrap.properties` @ line 89

Also totally removed from other per-environment files.

If we still want to do that, please see #4708.

Reply:
This is 0.35 so it's fine to be on in previewnet. 
There's no change to side cars so things should be fine.
We'll plan to turn that on in 0.36

Reply:
ok